doc/structural.xxml

```diff
@@ -24,6 +24,7 @@
 <!-- doxrox-include igraph_distances_johnson -->
 <!-- doxrox-include igraph_get_shortest_paths -->
 <!-- doxrox-include igraph_get_shortest_path -->
+<!-- doxrox-include igraph_get_k_shortest_paths -->
```

Comment:
Can you move this to just before `get_all_simple_paths()`? It should not interrupt a block of closely related functions.